src/views/scroll-editor/artefact-image-group.vue

```diff
@@ -89,9 +86,55 @@ import {
 } from './operations';
 import { Placement } from '../../utils/Placement';
 import IIIFImageComponent from '@/components/images/IIIFImage.vue';
-import { InterpretationRoi } from '@/models/text';
+import { InterpretationRoi, SignInterpretation } from '@/models/text';
 import RoiLayer from '../artefact-editor/roi-layer.vue';
-import { forEach } from 'mathjs';
+
+class DisplayableSign {
+    public id: number;
+    public character: string;
+    public boundingBox: BoundingBox;
+    public yOffset: number = 0;
+
+    public constructor( si: SignInterpretation,
+                        rois: InterpretationRoi[],
+                        yOffset: number ) {
+        if (!si.character) {
+            throw new Error('DisplayedSign with no character');
+        }
+
+        if (!rois || !rois.length) {
+            throw new Error('DIsplayedSign with no rois');
+        }
+
+        this.id = si.id;
+        this.character = si.character;
+        this.yOffset = yOffset;
+
+        this.boundingBox = this.calculateBoundingBox(rois);
+    }
+
+    private calculateBoundingBox(rois: InterpretationRoi[]): BoundingBox {
+        const boundingBoxes: BoundingBox[] = [];
+
+        for (const roi of rois) {
+            const bbox = roi.shape.getBoundingBox();
+            bbox.x = roi.position.x ;
+            bbox.y = roi.position.y + this.yOffset;
+
+            boundingBoxes.push(bbox);
+        }
+
+        return BoundingBox.combine(boundingBoxes);
+    }
+
+    public get svgTransform() {
+        const midX = this.boundingBox.x ; // removed width, prevent shift right
+        const midY = this.boundingBox.y - this.yOffset / 2 + this.boundingBox.height  / 2;
```

Comment:
I feel a bit dumb it took me so long to figure this out: change the line to `const midY = this.boundingBox.y - this.yOffset;`